### PR TITLE
Change to slash commands

### PR DESCRIPTION
**Pull request by pandabear189**
change normal commands to slash

**FILL OUT THE FIELDS BELOW!**

**Discord Username and Tag**
pandabear#8652

**Describe the Changes You've Created**
Changing commands to be slash commands

**Where are these changes going?**
GLobally

**Confirm that you have done the following:**
- [x] Created a new folder for your bot under utils/bots/~
- [x] Moved any events under the events folder. 
- [x] Properly Documented the code changes
- [x] DM'd Space for a new Documentation Page (for new commands)
- [x] Tested the New Files on your own and you confirm that the changes are bug free.

**Additional Context**
Add any other information about the Pull Request here.

### CODE REVIEW

**Comment by pandabear189** [comment]
Solves 1/2 of #107

File: utils/bots/TutorBot/cogs/TutorStaff.py [hunk 1]
```diff
@@ -75,97 +82,32 @@ async def remove(self, ctx, id):
...
             embed = discord.Embed(
                 title="Invalid Session",
                 description="This session does not exist, please check the ID you've provided!",
                 color=discord.Color.red(),
             )
-            await ctx.send(embed=embed)
-
-    @commands.command()
-    @commands.has_role("Tutor")
-    async def schedule(
```
Comment (Space-Turtle0) [inline utils/bots/TutorBot/cogs/TutorStaff.py:90]: If you're going to delete this, perhaps check the schedule slash command in main.py (~line 209) and update the guild id and permissions there?

**Review by Space-Turtle0 — CHANGES_REQUESTED** [review]
Nice PR! Check the comment on the schedule command though, that's the only thing that needs changing.

File: utils/bots/TutorBot/cogs/TutorStaff.py (hunk 1 above)
Comment (pandabear189) [reply utils/bots/TutorBot/cogs/TutorStaff.py:90]: why was this command limited to tutors but the other wasnt?

**Comment by Space-Turtle0** [comment]
Also, we're probably going to need to add a message in the error handler about commands being ported over to slash commands. Kind of a user notice, maybe add something like this? It just needs to point over to the website and I can create a page about the changes.

```py
elif isinstance(error, (commands.CommandNotFound, commands.errors.CommandNotFound)):
        cmd = ctx.invoked_with
        cmds = [cmd.name for cmd in bot.commands]
        matches = get_close_matches(cmd, cmds)
        await ctx.send("Some commands have moved over to slash commands, please check https://timmy.schoolsimplified.org for more updates!")

        if len(matches) > 0:
            return await ctx.send(
                f'Command "{cmd}" not found, maybe you meant "{matches[0]}"?'
            )
        else:
            return await ctx.send(
                f'Command "{cmd}" not found, use the help command to know what commands are available'
            )


```

File: utils/bots/TutorBot/cogs/TutorStaff.py (hunk 1 above)
Comment (Space-Turtle0) [reply utils/bots/TutorBot/cogs/TutorStaff.py:90]: 🤷, I'm pretty sure it was limited to the tutor server so we just assumed everyone on it was staff.

**Comment by pandabear189** [comment]
yea, that will def be necessary or we can do something similar with the did you mean this command where we list interpret for like... 
```py
[x.name for x in bot.application_commands]
```
and then see if its a slash command now and send a specific message for that

File: utils/bots/TutorBot/cogs/TutorStaff.py (hunk 1 above)
Comment (pandabear189) [reply utils/bots/TutorBot/cogs/TutorStaff.py:90]: oh i see

**Comment by pandabear189** [comment]
ok should be ready to merge

**Comment by Space-Turtle0** [comment]
Yeah, that too. Make it point to the website so I can create a more detailed explanation of the change.

**Review by Space-Turtle0 — APPROVED** [review]

**Review by Space-Turtle0 — CHANGES_REQUESTED** [review]
Could you change the URL to: https://timmy.schoolsimplified.org/#slash-command-port
That's the only change required and then I'll merge.

**Review by Space-Turtle0 — APPROVED** [review]

**Pull request merged**